website/docs/BasicInterfaces/SecretAgent.md

```diff
@@ -52,6 +52,7 @@ const SecretAgent = require('secret-agent');
   - renderingOptions `string[]`. Controls browser functionality.
   - userProfile `IUserProfile`. Previous user's cookies, session, etc.
   - showReplay `boolean`. Whether or not to show the Replay UI. Can also be set with an env variable: `SA_SHOW_REPLAY=true`.
+  - upstreamProxy `IUpstreamProxy`. A socks5 or http proxy url (and optional username) to use for all HTTP requests in this session. Dns over Tls requests will also use this proxy, if provided. Auth should be formatted as `user:password`. Http proxy auth will be converted to base64 basic authentication.
```

Comment:
When it says, "and optional username" does that mean "and optional auth"? Or is it referring to username being in the url?

Reply:
Should say auth. The proxy configurations/docs will sometimes refer to this as username. I'm mixing terms I shouldn't be. Will correct